lib/mapObjectConstructors/AObjectTypeHandler.cpp

```diff
@@ -96,6 +96,18 @@ void AObjectTypeHandler::init(const JsonNode & input)
 	else
 		aiValue = static_cast<std::optional<si32>>(input["aiValue"].Integer());
 
+	// TODO: Define properties, move them to actual object instance
+	if(input["blockVisit"].isNull())
+		blockVisit = false;
+	else
+		blockVisit = input["blockVisit"].Bool();
+
+	if(input["removable"].isNull())
+		removable = false;
+	else
+		removable = input["removable"].Bool();
+
+
```

Comment:
Our JsonNode evaluates bool to false if entry is not present.
And second block is wrong - should have been inverted
```suggestion
	// TODO: Define properties, move them to actual object instance
	blockVisit = input["blockVisit"].Bool();
	removable = input["removable"].Bool();

```